base_import_module_group/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "auto-install": True,
+    "auto_install": True,
```

Comment:
Actually, no. Why should this module be `auto_install` anyway? Let's just remove the whole key instead.

Reply:
Seems reasonable: no-one will be missing a feature that doesn't work